src/yoroi-wallets/Wallet.ts

```diff
@@ -307,37 +306,6 @@ export class Wallet {
     this.notify({type: 'state', state: this.state})
   }
 
-  canGenerateNewReceiveAddress() {
```

Comment:
why did you move this?

Reply:
The methods has been moved from `walletManager` too. There, one method was invoking `wallet.save()`, considering `Wallet` class is going to be removed I tried to avoid to use `super` again.

Reply:
I think it's correct for the impl to be inside the specialized class, but I would expect to have these methods in the wallet interface.